Eliminate parsing chapter (#1128)
hadley/r4ds: numbers.qmd

library(nycflights13)
```

## Making numbers

> In most cases, you'll get numbers already recorded in one R's numeric types, integer or double.

Reviewer: ```suggestion
In most cases, you'll get numbers already recorded in one of R's numeric types: integer or double.
```

> In some cases, however, you'll encounter them as strings, possibly because you've created the by pivotting from column headers or something has gone wrong in your data import process.

Reviewer: ```suggestion
In some cases, however, you'll encounter them as strings, possibly because you've created them by pivoting from column headers or something has gone wrong in your data import process.
```

## Summary

You're already familiar with many tools for working with numbers, and after reading this chapter you now know how to use them in R.

> You've also learned a handful of useful general transformations that are commonly, but not exclusively, applied to numeric vector like ranks and offsets.

Reviewer: ```suggestion
You've also learned a handful of useful general transformations that are commonly, but not exclusively, applied to numeric vectors like ranks and offsets.
```
